lib/agoneum/account/user.ex

```diff
@@ -4,6 +4,12 @@ defmodule Agoneum.Account.User do
   import Ecto.Changeset
   alias Agoneum.Account.User
 
+  @required_fields ~w(email name)a
+  @optional_fields ~w(active admin password)a
```

Comment:
You very much don't want to include admin here in all changesets as you unlock the ability to make a user an admin through an edit user form.

The way I've done this is the main `changeset` is allowed to change admin, but the only thing that actually uses it is locked behind admin. Your registration_changeset or basically any other user_changeset should not have the admin field fed to cast at all.